pkg/resources/webhook.go

```diff
@@ -0,0 +1,281 @@
+package resources
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/gthesheep/terraform-provider-dbt-cloud/pkg/dbt_cloud"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+var (
+	eventTypes = []string{
+		"job.run.completed",
+		"job.run.started",
+		"job.run.errored",
+	}
+)
+
+var webhookSchema = map[string]*schema.Schema{
+	"webhook_id": &schema.Schema{
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Webhooks ID",
+	},
+	"name": &schema.Schema{
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Webhooks Name",
+	},
+	"description": &schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		Default:     "",
+		Description: "Webhooks Description",
+	},
+	"client_url": &schema.Schema{
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Webhooks Client URL",
+	},
+	"event_types": &schema.Schema{
+		Type:        schema.TypeList,
+		Required:    true,
+		Description: "Webhooks Event Types",
+		Elem: &schema.Schema{
+			Type:         schema.TypeString,
+			ValidateFunc: validation.StringInSlice(eventTypes, false),
+		},
+	},
+	"job_ids": &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "List of job IDs to trigger the webhook, An empty list will trigger on all jobs",
+		Elem: &schema.Schema{
+			Type: schema.TypeInt,
+		},
+	},
+	"active": &schema.Schema{
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     true,
+		Description: "Webhooks active flag",
+	},
+	"hmac_secret": &schema.Schema{
+		Type:        schema.TypeString,
+		Computed:    true,
+		Sensitive:   true,
+		Description: "Webhooks active flag",
+	},
+	"http_status_code": &schema.Schema{
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Webhooks HTTP Status Code",
+	},
+	"account_identifier": &schema.Schema{
```

Comment:
is this the dbt Cloud account ID? Think it exists on the Client which could save having it here?